website/docs/r/postgresql_flexible_server_active_directory_administrator.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_postgresql_flexible_server_active_directory_administrator"
+description: |-
+  Manages an Active Directory administrator on a PostgreSQL Flexible server
+---
+
+# azurerm_postgresql_flexible_server_active_directory_administrator
+
+Allows you to set a user or group as the AD administrator for an PostgreSQL Flexible server in Azure
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_postgresql_flexible_server" "example" {
+  name                         = "example-psqlserver"
+  resource_group_name          = azurerm_resource_group.example.name
+  location                     = azurerm_resource_group.example.location
+  version                      = "12"
+  administrator_login          = "4dm1n157r470r"
+  administrator_login_password = "4-v3ry-53cr37-p455w0rd"
+  sku_name                     = "GP_Standard_D2s_v3"
+  zone                         = "2"
+
+  auth_config {
+    active_directory_auth_enabled = true
+    tenant_id                     = data.azurerm_client_config.current.tenant_id
+  }
+}
+
+resource "azurerm_postgresql_flexible_server_active_directory_administrator" "example" {
+  server_name         = azurerm_postgresql_server.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  object_id           = data.azurerm_client_config.current.object_id
+  principal_name      = "example-sp"
+  principal_type      = "ServicePrincipal"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `server_name` - (Required) The name of the PostgreSQL Flexible Server on which to set the administrator. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group for the PostgreSQL server. Changing this forces a new resource to be created.
+
+* `object_id` - (Required) The object ID of a user, service principal or security group in the Azure Active Directory tenant set as the Flexible Server Admin.
+
+* `tenant_id` - (Required) The Azure Tenant ID.
+
+* `principal_name` - (Required) The name of Azure active directory principal.
+
+* `principal_type` - (Required) The type of Azure active directory principal. Possible values are `Group`, `ServicePrincipal` and `User`.
```

Comment:
It looks like these are ForceNew in the schema, so we should add Changing this forces a new resource to be created. to these descriptions.